### DISCUSSION TITLE
More explicit eject process for CD set?

### DISCUSSION THREAD

**Discussion by henris42** [erichelgeson/BlueSCSI-v1#249] [Ideas]
Would it be possible to utilize LED and button to make the CD set eject/insert process more visible?

For example if CD is in ejected state, LED would light up, and you'd use a button to "insert" next one in the set?

I think this could make things easier with many installers in very old systems.

**Comment by erichelgeson** [comment]
Hi! - BlueSCSI v1 doesn't have a CD Set feature - v2 does and does have the ability to hook up a button. If someone wanted to back port that feature I'd gladly review it.
